app/views/site/genome/_genome.html.erb

```diff
@@ -3,7 +3,9 @@ Minimal DOM scaffolding and server-side data for genomic visualizations.
 See partials referenced below and scp-igv.js for larger bodies of code.
 -->
 <script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
+  console.log('in _genome.html.erb')
   if (typeof accessToken === 'undefined') {
+    console.log('accessToken is undefined')
```

Comment:
remove console.log

Reply:
Good catch, fixed in 2636780.  This template was inlined into `app/views/site/_study_visualize.html.erb `.